nixos/tests/sssd-ldap.nix

```
@@ -91,6 +91,8 @@ in import ./make-test-python.nix ({pkgs, ...}: {
     machine.start()
     machine.wait_for_unit("openldap.service")
     machine.wait_for_unit("sssd.service")
+    if "${testUser}" not in machine.succeed("getent passwd ${testUser} || true"):
```

```suggestion
    if "${testUser}" not in machine.execute("getent passwd ${testUser}")[1]:
```

Since you don't care about the exit status anyway.